cohttp-lwt/src/s.ml

```diff
@@ -106,12 +106,37 @@ module type Server = sig
 
   type conn = IO.conn * Cohttp.Connection.t
 
+  type response_action =
+    (* The connection is not closed in the [`Expert] case until the [unit
```

Comment:
Can we add a proper docstring with the rationale for the response action and the content of this comment?

Reply:
Good point -- added. Let me know what you think.

Reply:
Thanks. They look great